app/src/main/java/chat/rocket/android/RocketChatApplication.java

```diff
@@ -23,6 +23,7 @@
   public void onCreate() {
     if (BuildConfig.DEBUG) {
```

Comment:
Looks like the CircleCI doesn't take in consideration the `if (BuildConfig.DEBUG) {...}`. Because of it the build is failing (but work properly). Can someone answer me if is there an option/configuration on CircleCI to solve it?

Create a src/debug folder only to solve it, having the `if (BuildConfig.DEBUG) {...}`, is boring.
No warnings w/ TravisCI.